bool initialized_
int8 data